usecases/cluster/delegate.go

```diff
@@ -234,17 +238,20 @@ func (d *delegate) delete(node string) {
 // sortCandidates by the amount of free space in descending order
 //
 // Two nodes are considered equivalent if the difference between their
-// free spaces is less than 4KB.
+// free spaces is less than 32MB.
 // The free space is just an rough estimate of the actual amount.
-// The Lower bound 4KB helps to mitigate the risk of selecting same set of nodes
+// The Lower bound 32MB helps to mitigate the risk of selecting same set of nodes
 // when selections happens concurrently on different initiator nodes.
 func (d *delegate) sortCandidates(names []string) []string {
+	rand.Shuffle(len(names), func(i, j int) { names[i], names[j] = names[j], names[i] })
+
 	d.Lock()
 	defer d.Unlock()
 	m := d.Cache
 	sort.Slice(names, func(i, j int) bool {
-		return (m[names[j]].Available >> 12) < (m[names[i]].Available >> 12)
+		return (m[names[j]].Available >> 25) < (m[names[i]].Available >> 25)
```

Comment:
why we bit shift here ? won't have the same results as without ? 

```go
sort.Slice(names, func(i, j int) bool {
    return (m[names[j]].Available) < (m[names[i]].Available)
})
```

Reply:
No since we don't have precise information about the current available disk space, so we need to tolerate differences in available disk space (Available >> 25, 32MB); otherwise, all tenants would end up on the same node.